corehq/apps/userreports/ui/fields.py

```diff
@@ -57,4 +60,6 @@ def validate(self, value):
         if value in self.null_values and self.required:
             raise forms.ValidationError(self.error_messages['required'])
         if self.expected_type and not isinstance(value, self.expected_type):
-            raise forms.ValidationError(_('Expected {} but was {}'.format(self.expected_type, type(value))))
+            raise forms.ValidationError(
+                _('Expected {} but was {}'.format(self.expected_type.__name__, type(value).__name__))
```

Comment:
I like that you've used `__name__` to make type names more readable.

`format` should be outside your call to `_`:
```suggestion
                _('Expected {} but was {}').format(self.expected_type.__name__, type(value).__name__)
```